MAINTAINERS.yml

```diff
@@ -1347,6 +1347,16 @@ Filesystems:
   labels:
     - "area: File System"
 
+"Filesystems: FatFs":
+  status: maintained
+  maintainers:
+    - ox11
+  files:
+    - subsys/fs/fat_fs.c
```

Comment:
I see no reason to pass Maintainership on this file, it is not even touched in this PR. Remove it from list.

Reply:
Yes, you are right.